package.json

```diff
@@ -30,6 +30,14 @@
       }
     }
   },
+  "providedServices": {
+    "tree-view": {
+      "description": "Get treeView Instance",
```

Comment:
Rewrite this to something more descriptive and helpful to developers; perhaps something like:
"A tree-like view of directories and files"

/cc @atom/feedback